hackathon_site/registration/models.py

```diff
@@ -117,27 +135,59 @@ class Application(models.Model):
 
     q1 = models.TextField(
```

Comment:
Since we're generating migrations anyway, we might as well rename all of the fields from `q1` etc to something more descriptive:

- `what_hackathon_experience`
- `why_participate`
- `what_technical_experience`
- `referral_source`